src/olympia/addons/admin.py

```diff
@@ -194,9 +194,22 @@ def change_view(self, request, object_id, form_url='', extra_context=None):
                     url += '?' + request.GET.urlencode()
                 return http.HttpResponsePermanentRedirect(url)
 
-        return super(AddonAdmin, self).change_view(
-            request, object_id, form_url, extra_context=None,
+        response = super(AddonAdmin, self).change_view(
+            request, object_id, form_url, extra_context=extra_context
         )
+        if request.method == 'GET':
```

Comment:
mmm, tbh I don't like tinkering with `TemplateResponse`.

If you override `render_change_form` instead we have the `obj` reference passed in as a parameter and you can add it to `context`

Reply:
ah yes, that's much better, thanks!